source/backend/scheduler/Startup.cs

```diff
@@ -267,6 +267,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddHangfireServer(options =>
             {
                 options.Queues = hangfireOptions.Queues;
+                options.WorkerCount = 1;
```

Comment:
Runs the scheduled document tasks serially, for now. This prevents very large files from being uploaded twice before PIMS can set the processing flag due to long clamav processing time.